Validation/SiPixelPhase1TrackingParticleV/src/SiPixelPhase1TrackingParticleV.cc

```diff
@@ -51,6 +51,7 @@ void SiPixelPhase1TrackingParticleV::analyze(const edm::Event& iEvent, const edm
   edm::Handle<TrackingParticleCollection>  TruthTrackContainer;
   iEvent.getByToken( vec_TrackingParticle_Token_, TruthTrackContainer );
   const TrackingParticleCollection *tPC   = TruthTrackContainer.product();
+  std::vector<std::pair<unsigned int, const PSimHit *>> trackIdToHitPtr_;
```

Comment:
Please change name to just `trackIdToHitPtr` since we reserve a following underscore for member data names.

Reply:
Will do!

Reply:
Also, be sure to do 
`curl https://cmssdt.cern.ch/SDT/code-checks/PR-21084/1718/git-diff.patch | patch -p1`
to pick up the changes needed to pass the code-checks.